config/config.go

```diff
@@ -843,6 +843,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.beintoo.endpoint", "https://ib.beintoo.com/um")
 	v.SetDefault("adapters.between.endpoint", "http://{{.Host}}.betweendigital.com/openrtb_bid?sspId={{.PublisherID}}")
 	v.SetDefault("adapters.bidmachine.endpoint", "https://{{.Host}}.bidmachine.io")
+	v.SetDefault("adapters.bidmyadz.endpoint", "http://endpoint.bidmyadz.com/c0f68227d14ed938c6c49f3967cbe9bc")
```

Comment:
Please confirm the uuid in the endpoint refers to the Prebid ecosystem as a whole and not an specific host, such as AppNexus/Xandr.

Reply:
This hash was created specifically for prebid traffic. It can be used by multiple partners.